src/main/scala/mesosphere/marathon/core/readiness/impl/SprayReadinessCheckExecutor.scala

```diff
@@ -0,0 +1,81 @@
+package mesosphere.marathon.core.readiness.impl
+
+import akka.actor.ActorSystem
+import akka.util.Timeout
+import mesosphere.marathon.core.readiness.ReadinessCheckExecutor.ReadinessCheckSpec
+import mesosphere.marathon.core.readiness.{ HttpResponse, ReadinessCheckExecutor, ReadinessCheckResult }
+import org.slf4j.LoggerFactory
+import rx.lang.scala.Observable
+import spray.client.pipelining._
+import spray.http.HttpHeaders.`Content-Type`
+import spray.http.{ HttpResponse => SprayHttpResponse, _ }
+
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+import scala.util.Success
+import scala.util.control.NonFatal
+
+/**
+  * A Spray-based implementation of a ReadinessCheckExecutor.
+  */
+private[readiness] class SprayReadinessCheckExecutor(support: SprayReadinessCheckExecutor.Support)
```

Comment:
Can we rename this to `ReadinessCheckExecutorImpl` to have the same structure as in other modules and easier code navigation?